src/pudl/metadata/resources/ferc1.py

```diff
@@ -1686,9 +1686,56 @@
         "etl_group": "outputs",
         "field_namespace": "ferc1",
     },
+    "out_ferc1__detailed_income_statements": {
+        "description": (
+            "This table contains granular accounting data from utilities' income statements."
+            "This table is derived from four FERC Form 1 accounting tables with nested calculations:\n"
+            " - `core_ferc1__yearly_income_statements_sched114`\n"
+            " - `core_ferc1__yearly_depreciation_summary_sched336`\n"
+            " - `core_ferc1__yearly_operating_expenses_sched320`\n"
+            " - `core_ferc1__yearly_operating_revenues_sched300`\n"
+            "We reconciled the nested calculations within these tables and then identified the "
+            "most granular data across the tables.\n"
+            "We applied slight modifications to two columns (utility_type & plant_function)"
+            "as compared to the originally reported values in our core tables. "
+            "The modifications were applied to either provide more specificity (i.e. we converted"
+            "some `total` utility_type's into `electric`) or to condense similar categories "
+            "for easier analysis (i.e. creating a `hydraulic_production` plant_function by "
+            "combining `hydraulic_production_conventional` and `hydraulic_production_pumped_storage`\n"
+            "See ``pudl.output.ferc1.Exploder`` for more details. This table was made entirely from "
+            "support and direction of RMI."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "utility_id_ferc1",
+                "xbrl_factoid",
+                "utility_type",
+                "plant_function",
+                "revenue_requirement_technology",
+                "dollar_value",
+                "in_revenue_requirement_review",
+                "revenue_requirement_category",
+                "table_name",
+            ]
+            # This table would have a composite primary_key of the first 5 columns listed here, but
+            # there are expected nulls in utility_type and plant_function. and ~60 records from 1998
+            # and before would have dupes with those pks.
```

Comment:
i noticed in the todo list i wrote for myself long ago that i should convert this into an asset check like `check_pks`